persistence/src/commonMain/db_user/com/wire/kalium/persistence/Clients.sq

```diff
@@ -120,3 +120,15 @@ BEGIN
         WHERE Client.id = new.id
     );
 END;
+
+CREATE TRIGGER inserNewSelfClient
+AFTER INSERT ON Client
+WHEN new.user_id = (SELECT SelfUser.id FROM SelfUser LIMIT 1)
+BEGIN
+    INSERT INTO NewClient(id, device_type, registration_date, model)
+    VALUES(new.id, new.device_type, new.registration_date, new.model)
+    ON CONFLICT (id) DO UPDATE SET
+    device_type = coalesce(excluded.device_type, device_type),
+    registration_date = coalesce(excluded.registration_date, registration_date),
+    model = coalesce(excluded.model, model);
+END;
```

Comment:
I also wonder how this will work when logging in for the first time. 
My guess it will insert all of the user other clients as new and they wil always have the the dialog shown after a new login